### PR TITLE
Added ability to change zoom using Ctrl + Scroll in Document Viewer

### PR DESCRIPTION
**Pull request by cardionaut**
I have added the ability to change the zoom level using Ctrl + Scroll in the PDF Document Viewer.
Additionally, I have limited the zoom-out to ~1 page by setting a minWidth, as an unlimited zoom-out led to following problems:
- Occasional display errors (e.g. no text displayed for certain pages)
- No content can be seen if zoomed out too far, might be unclear to the user what happened and that zooming in will fix problem

The Document Viewer is still not very user friendly in my opinion and I am not sure how much further development time should be invested in its current state.
It might be a better option to get rid of it alltogether and always use the "Open file (F4)" option when a user wants to open a PDF.

### Mandatory checks

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/gui/documentviewer/DocumentViewerControl.java [hunk 1]
```diff
@@ -91,7 +102,8 @@ private void updateCurrentPage(ObservableList<DocumentViewerPage> visiblePages)
                 // Successful hit
                 inMiddleOfViewport = Optional.of(hit.getCell());
             }
-        } catch (NoSuchElementException exception) {
+        } catch (
```
Comment (calixtus) [inline src/main/java/org/jabref/gui/documentviewer/DocumentViewerControl.java:105]: Currently IntelliJ has Problems with the catch blocks. It always creates a new line here on auto formatting, we are investigating. Please remove the newline manually.

**Review by calixtus — CHANGES_REQUESTED** [review]
Very cool change. Just one little code style nitpick. Then it's good to go in my eyes.

**Review by Siedlerchr — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - We added the possibility to redownload files that had been present but are no longer in the specified location. [#10848](https://github.com/JabRef/jabref/issues/10848)
 - We added the citation key pattern `[camelN]`. Equivalent to the first N words of the `[camel]` pattern.
 - We added ability to export in CFF (Citation File Format) [#10661](https://github.com/JabRef/jabref/issues/10661).
+- We added the ability to zoom in and out in the document viewer using <kbd>Ctrl</kbd> + <kbd>Scroll</kbd>.
```
Comment (Siedlerchr) [inline CHANGELOG.md:27]:
```suggestion
- We added the ability to zoom in and out in the document viewer using <kbd>Ctrl</kbd> + <kbd>Scroll</kbd> [#10964](https://github.com/JabRef/jabref/pull/10964).
```

Comment (koppor) [inline CHANGELOG.md:27]: We linked the PRs (if there is no issue)

```suggestion
- We added the ability to zoom in and out in the document viewer using <kbd>Ctrl</kbd> + <kbd>Scroll</kbd>. [#10964](https://github.com/JabRef/jabref/pull/10964)
```

**Comment by koppor** [comment]
I fully agree that the internal PDF viewer is a mess. The discussions are long ongoing. See https://sourceforge.net/p/jabref/feature-requests/667/ and some other feature request at https://sourceforge.net/p/jabref/discussion/318824/thread/4a6a1d4d/. It is more about convenience for the users not to have changing apps. For me, I would say: We should use existing apps, because we cannot implement a good PDF viewer on our own.

Maybe, we should investigate in a "proper" connection to existing PDF viewers. Similar to LaTeX editors do it.

I would first start with SumatraPDF - https://www.sumatrapdfreader.org/docs/Command-line-arguments. Then with Acrobat Reader? (https://stackoverflow.com/a/9554528/873282). I thought, Acrobat would have some specific communication infrastructure (DDX?), but I do not find it.

**Review by koppor — COMMENTED** [review]
Works great. One small nitpick (to reduce load on my side ^^)

**Comment by koppor** [comment]
In case, we go to keep the viewer internally - we should check out

- pdf.js - recommended at https://stackoverflow.com/a/42040344/873282
- Play around with https://github.com/dlsc-software-consulting-gmbh/PDFViewFX

However, the most important part is to be able to highlight words and add text comments. Not sure, which framework will support that.

**Comment by koppor** [comment]
@cardionaut I just committed my CHANGELOG.md update to reduce load on your side ^^

File: CHANGELOG.md (hunk 2 above)
Comment (Siedlerchr) [reply CHANGELOG.md]: That's what I literally did yesterday as well, but my changelog change was not commited.... 😭

**Review by Siedlerchr — APPROVED** [review]

**Comment by cardionaut** [comment]
> @cardionaut I just committed my CHANGELOG.md update to reduce load on your side ^^

@koppor Thanks!
I am still new to this, thought that resolving would automatically apply the changes, sorry :)

**Review by calixtus — APPROVED** [review]

**Pull request merged**